<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrainingResultView</class>
 <widget class="QWidget" name="TrainingResultView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>928</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_20" rowstretch="0" columnstretch="1,0,1,0,1">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox_10">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Accuracy Curve</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="Canvas" name="graphicsView_accuracycurve">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="3" colspan="2">
      <widget class="QGroupBox" name="groupBox_12">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Most Misclassified Images</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="ImageGallery" name="imageGallery_mostMisclassifiedImages" native="true"/>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QGroupBox" name="groupBox_11">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Confusion Matrix</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="Canvas" name="graphicsView_confusionmatrix">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="renderHints">
           <set>QPainter::Antialiasing|QPainter::LosslessImageRendering|QPainter::TextAntialiasing</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Canvas</class>
   <extends>QGraphicsView</extends>
   <header>canvas.h</header>
  </customwidget>
  <customwidget>
   <class>ImageGallery</class>
   <extends>QWidget</extends>
   <header>imagegallery.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
